pages/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,81 @@
+            bg-white w-full py-2.5 px-0`}
+          >
+            <li className="laptop:block laptop:float-left laptop:py-3 laptop:px-5 hidden float-none" >
+              <Link className='laptop:m-0 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:p-0 laptop:no-underline laptop:hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="">
+                <Image
+                  src={REALDEVSQUAD_IMAGE}
+                  alt="home nav logo"
+                  height={55}
+                  width={55}
+                />
+              </Link>
+            </li>
+            <li className="laptop:float-left laptop:hidden block float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-2.5 px-10 text-[#1d1283]' href="http://realdevsquad.com/">Home</Link></li>
+            <li className="laptop:float-left float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="https://welcome.realdevsquad.com/">Welcome</Link></li>
+            <li className="laptop:float-left float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="https://www.realdevsquad.com/events">Events</Link></li>
+            <li className="laptop:float-left float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="https://members.realdevsquad.com/">Members</Link></li>
+            <li className="laptop:float-left float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="https://crypto.realdevsquad.com/">Crypto</Link></li>
+            <li className="laptop:float-left float-none"><Link className='laptop:m-2.5 block laptop:rounded-lg laptop:text-white laptop:text-center laptop:py-5 laptop:px-4 laptop:no-underline hover:text-[#49a82e] py-[10px] px-[40px] text-[#1d1283]' href="https://status.realdevsquad.com/">Status</Link></li>
+            <li className="laptop:block laptop:float-right hidden float-none">
+              <Link className='laptop:m-[22px] block laptop:rounded-lg laptop:text-white laptop:text-center laptop:p-0 laptop:no-underline laptop:hover:text-[#49a82e] py-2.5 px-10 text-[#1d1283]'
+                href="#"
+              >
+                <button className="rounded-md p-[5px] bg-[#1d1283] text-white cursor-pointer border-2 border-solid border-white flex justify-around">

```

Comment:
please add an `onClick `here which has an `arrow function`  and when clicked throws error saying function not declared